spec/features/staff/users_can_create_a_transaction_spec.rb

```diff
@@ -129,4 +129,48 @@
       expect(page).to have_content "123000000"
     end
   end
+
+  context "date validations" do
```

Comment:
I wanted them to have a separate feature to check the text appears on the page. It's not correct to add them at line 33 because those fields are mandatory, and these date fields are not mandatory but show an error message if filled incorrectly. I'd prefer to keep them unless there's a strong case to get rid?

Reply:
Yeah, I wouldn't look for them at line 33 either.

Okay cool. I guess for me this gives me confidence Rails is working (showing the error) which I would question a feature for. If it gives you confidence I am for it! 👍